aws/aws-how-to/instances/automatically-setup-multiple-nics.rst

```diff
@@ -0,0 +1,55 @@
+Set up automated network configuration for instances with multiple NICs
+=======================================================================
+
+Ubuntu instances running on AWS can be configured to automatically configure source routing on multiple NICs via cloud-init, starting with `v24.1`_, on different `events`_, as:
+
+* First boot (enabled by default)
+* Subsequent boots
+* Hotplug NIC additions and removals
+
+.. note::
+
+   On Ubuntu Noble and newer, hot-plugged NICs are configured by default.
+
+Enable additional network configuration events
```

Comment:
If we change the two section titles (this one and the next one) to the following, will it be easier to understand?

- Enable configuration during instance creation
- Enable configuration on running instances